lib/saml/user_attributes/ssoe.rb

```diff
@@ -50,6 +54,19 @@ def email
       end
 
       ### Identifiers
+      def idme_uuid
+        return safe_attr('va_eauth_uid') if sign_in[:service_name] == 'idme'
```

Comment:
this whole `sign_in` object is pretty clunky; i'm wondering if there's anything in the assertion itself that we can use to distinguish. If you look at the factories under `spec/factors/saml_attributes.rb`, there's examples  of all  the login variations, there are some CSP attributes

Reply:
I saw the `va_eauth_csid` attribute, but I've been unable to test any MHV / DS Logon accounts so I wasn't sure that it would be available every time.

Reply:
Those factories should be current/accurate w.r.t. what traits are available, and have instances of each credential variation. `csid` looks to be equal to `idme` for any case where ID.me is brokering the authentication. I'll have data on what it looks like for inbound existing sessions pretty soon.